ui/desktop/app/templates/scopes/scope/projects/targets/loading.hbs

```diff
@@ -0,0 +1,14 @@
+<div class='search-filtering-loading'>
+  <div class='search-filtering-loading-title'>
+    <FlightIcon @name='loading' @size='24' />
+    <Hds::Text::Display @tag='h3'>
+      {{t 'states.loading.title'}}
+    </Hds::Text::Display>
+  </div>
+  <Hds::Text::Body @tag='p'>
+    {{t
+      'states.loading.description'
+      resource=(t 'resources.target.title_plural')
+    }}
+  </Hds::Text::Body>
+</div>
```

Comment:
non-blocking:
Could we use [Hds::ApplicationState](https://helios.hashicorp.design/components/application-state?tab=code#empty-state-with-header-icon) for here and the `/sessions/loading` template?

Reply:
ahh good call, totally forgot about the existence of this component, I'll update it